codechecker_lib/generic_package_context.py

```diff
@@ -64,24 +66,40 @@ def set_env(self, env_vars):
         self.ld_preload = os.environ.get(env_vars['ld_preload'])
         self.ld_lib_path = env_vars['env_ld_lib_path']
 
-    def __get_version(self, version_file):
+    def __set_version(self):
         """
         Get the package version from the version config file.
         """
         try:
-            with open(version_file, 'r') as vfile:
+            with open(self.version_file, 'r') as vfile:
                 vfile_data = json.loads(vfile.read())
 
             package_version = vfile_data['version']
-            db_version = vfile_data['db_version']
+            package_build_date = vfile_data['package_build_date']
+            package_git_hash = vfile_data['git_hash']
+            database_version = vfile_data['db_version']
```

Comment:
As far as I can see the `version.json` file contains only the `version` and `db_version` attributes. How do `package_build_date` and `git_hash` get there?